model/trace/database.yaml

```diff
@@ -26,9 +28,15 @@ groups:
     brief: This group defines the attributes used to perform database client calls.
     attributes:
       - ref: db.query.text
+        note: >
+          For sanitization see [Sanitization of `db.query.text`](../../docs/database/database-spans.md#sanitization-of-dbquerytext)
+
+          For batch operations, if the individual operations are known to have the same query text
+          then that query text SHOULD be used, otherwise all of the individual query texts SHOULD be concatenated
+          with separator `; ` or some other database system specific separator if more applicable.
         requirement_level:
           recommended: >
-            SHOULD be collected by default only if there is sanitization that excludes sensitive information.
+            See [Sanitization of `db.query.text`](../../docs/database/database-spans.md#sanitization-of-dbquerytext).
```

Comment:
what happens if we inherit note and requirement_level from `attributes.db.client.minimal`?

Reply:
In short: for note, we'd loose the part that talks about batch operations, but for `requirement_level`, we could probably do that. A few things inherit from `trace.db.common.query_and_collection` (e.g. cassandra, sql, cosmos), so the effect of doing it for `requirement_level` will be that all those will have a longer text for `requirement_level` talking about parameterized and Non-parameterized queries. But I think that's fine.

Regarding `note`: there is also a note in the registry, which we seem to overwrite everywhere. I don't really understand yet why. Wouldn't it be easier to just handle the note for db.query.text in the registry and just inherit that everywhere? So far I don't see the logic of differentiating that note for the different ids.

Reply:
I moved the general note to the registry and consolidated it there - I really wonder why it was overwritten [here](https://github.com/open-telemetry/semantic-conventions/blob/main/model/trace/database.yaml#L16-L19). 

With this, we just inherit the note from the registry which now covers the link to sanitization, batching, and parameterized query.

Let me know if this is not an ok move.